src/uct/ib/rc/accel/rc_mlx5_iface.c

```diff
@@ -154,6 +154,8 @@ uct_rc_mlx5_iface_poll_tx(uct_rc_mlx5_iface_common_t *iface)
     uct_rc_mlx5_txqp_process_tx_cqe(&ep->super.txqp, cqe, hw_ci);
     ucs_arbiter_group_schedule(&iface->super.tx.arbiter, &ep->super.arb_group);
     uct_rc_mlx5_iface_update_tx_res(&iface->super, ep, hw_ci);
+    ucs_arbiter_dispatch(&iface->super.tx.arbiter, 1, uct_rc_ep_process_pending,
+                         NULL);
```

Comment:
why needed here? it is called in `uct_rc_mlx5_iface_update_tx_res`

Reply:
we removed it from `uct_rc_mlx5_iface_update_tx_res`